uploader.json

```diff
@@ -1426,20 +1426,6 @@
             ],
             "symbolic_name": "ceph_conf"
         },
-        {
-            "file": "/var/log/ceph/ceph.log",
```

Comment:
To match the spec in insights core this needs to be changed to: `"file": "/var/log/ceph/()ceph.log*"`